qa/tasks/cephadm.conf

```diff
@@ -3,6 +3,7 @@
 log_to_file = true
 log_to_stderr = false
 log to journald = false
+mon cluster log to file = true
```

Comment:
Does this serve the same purpose as https://github.com/ceph/ceph/pull/48539 ?

Reply:
We still need https://github.com/ceph/ceph/pull/48539 in case something else goes sideways.

@adk3798 Please prioritize adding necessary allowlists to cephadm suite -- it's what that PR is blocked on and why I have been unstaling it for months now.  Chris and I didn't want to merge something that would turn one of the major suites red, but there a limit to how long something like that can wait...

Reply:
> I couldn't confirm if this cephadm.conf change makes the ceph.log appear -- can you please help testing that?

@dvanders If it does, it would turn cephadm suite red because, unlike in other suites, there are no ignorelists there.

Reply:
@idryomov  At this point, if it's causing failures to be missed elsewhere, I'm okay with it being merged. I have a partially complete ignorelist PR and this will at least force me to finish it. I can deal with sifting through the failures for the time before I finish that as well.

Reply:
@adk3798 I have asked @chrisphoffman to move forward on https://github.com/ceph/ceph/pull/48539.

Reply:
> @adk3798 I have asked @chrisphoffman to move forward with #48539.

This hasn't happened.  I have unstaled #48539 again, but would suggest moving forward by picking up this PR and seeing if it actually makes ceph.log file appear and causes those ignorelist-related failures.

Reply:
> If it does, it would turn cephadm suite red

Adam is asking for another run, but it's definitely working:

https://pulpito.ceph.com/yuriw-2024-01-31_15:48:29-rados:cephadm-main-distro-default-smithi/

Let's get this merged!